pkg/controller/job/job_controller_test.go

```diff
@@ -375,7 +375,25 @@ func TestControllerSyncJob(t *testing.T) {
 			expectedPodPatches:      2,
 			expectedFailed:          2,
 		},
-
+		"more terminating pods than active": {
+			parallelism:  4,
+			completions:  1,
+			backoffLimit: 6,
+			initialStatus: &jobInitialStatus{
+				startTime: func() *time.Time {
+					now := time.Now()
+					return &now
+				}(),
+			},
+			activePods:              2,
+			failedPods:              0,
+			terminatingPods:         4,
+			podReplacementPolicy:    podReplacementPolicy(batch.Failed),
+			jobPodReplacementPolicy: true,
+			expectedTerminating:     ptr.To[int32](4),
+			controllerTime:          &referenceTime,
```

Comment:
I was getting failures because start time wasn’t set so I added this. Idk why start time wouldn’t be set without but it seems to depend on those parameters.